src/sdk/object_sdk.js

```diff
@@ -687,6 +687,7 @@ class ObjectSDK {
                 time: Date.now() - start_time,
                 op_name: `upload_part`,
                 error,
+                trigger_send: false,
```

Comment:
Not sure we actually want to avoid sending per each `upload_multipart`.
@dannyzaken @jackyalbo WDYT?

Reply:
@romayalon WDYT?

Reply:
why not? don't we want to trigger it on every file system write?

Reply:
@romayalon Initially it was made too many sent triggers, but now we accumulate 10 sec in any case...
I think that we can remove it.